scripts/lis_hyperv_platform/trigger_lis_perf_tests.ps1

```diff
@@ -0,0 +1,23 @@
+param(
```

Comment:
can you reuse \scripts\\lis_hyperv_platform\\trigger_perf_tests.ps1" for this purpose?

Reply:
fixed